src/loc.rs

```diff
@@ -65,9 +76,16 @@ impl Stats {
 
     #[inline(always)]
     pub fn lloc(&self) -> f64 {
+        // This metric counts the instruction lines present in the code
         self.lines.len() as f64
     }
 
+    #[inline(always)]
+    pub fn stat(&self) -> f64 {
+        // This metric counts all the statements present in the source code
+        self.logical_lines.values().sum::<u64>() as f64
```

Comment:
Names are confusing here... maybe something like statements_by_line (`stat` itself looks like stats for statistics)

Reply:
I'd rather to maintain a known acronym like the other `loc` metrics. What do you think of this mapping?

lloc -> ploc (Physical lines of code)
stat -> lloc (Logical Lines of Code)
 
With some comments to explain the acronyms addressed by this metric.